src/main/groovy/com/github/jrubygradle/GemUtils.groovy

```diff
@@ -112,6 +112,11 @@ class GemUtils {
 
                 systemProperties 'file.encoding' : 'utf-8'
             }
+            project.javaexec {
```

Comment:
@rtyler Now I understand. I replied from my mobile phone without looking at the code. It would be good if we can run it as one JVM process, but rather raise an additional issue so that we can look. At least we can have something now that works. I'm very happy (_read_ excited) with the `.jrubydir` inclusion.